src/main/java/com/conveyal/gtfs/loader/JdbcGtfsLoader.java

```diff
@@ -259,27 +258,30 @@ private void registerFeed (File gtfsFile) {
     /**
      * In GTFS feeds, all files are supposed to be in the root of the zip file, but feed producers often put them
      * in a subdirectory. This function will search subdirectories if the entry is not found in the root.
-     * It records an error if the entry is in a subdirectory.
+     * It records an error if the entry is in a subdirectory (as long as errorStorage is not null).
      * It then creates a CSV reader for that table if it's found.
+     *
+     * This method is static so that it can be reused outside of JdbcGtfsLoader for getting CSV readers for zip files
+     * outside of the load procedure.
```

Comment:
This comment makes me wonder if it may be more appropriate to have this method be within the Table class instead. Is that possible?